app/models/office.rb

```diff
@@ -1,3 +1,5 @@
+require 'uk_postcode'
```

Comment:
It doesn't work if you don't require the gem, so unless there's a way to auto-require that I don't know about ...